crypto/CMakeLists.txt

```diff
@@ -479,6 +479,21 @@ add_library(
   x509v3/v3_skey.c
   x509v3/v3_utl.c
   xts/xts.c
+  ../decrepit/bio/base64_bio.c
+  ../decrepit/blowfish/blowfish.c
+  ../decrepit/cast/cast.c
+  ../decrepit/cast/cast_tables.c
+  ../decrepit/cfb/cfb.c
+  ../decrepit/des/cfb64ede.c
+  ../decrepit/dh/dh_decrepit.c
+  ../decrepit/dsa/dsa_decrepit.c
+  ../decrepit/evp/dss1.c
+  ../decrepit/evp/evp_do_all.c
+  ../decrepit/obj/obj_decrepit.c
+  ../decrepit/rc4/rc4_decrepit.c
+  ../decrepit/ripemd/ripemd.c
+  ../decrepit/rsa/rsa_decrepit.c
+  ../decrepit/x509/x509_decrepit.c
```

Comment:
Should we be moving the source files from the `decrepit` directory to `crypto`?

Reply:
Keeping current directory structure makes `git blame` nicer. If the files are moved, we will need twice `git blame` to see the original commits.  
Previously, we had trouble looking for code commits because aws-lc moved all upstream code/files in a special folder.

Reply:
Will submit a change.

Reply:
If we moved the source files, my concern was if that would make us taking upstream commits from BoringSSL harder? I'm not quite sure if git is smart enough to detect the new changes in a new file location.